build_runner/README.md

```diff
@@ -83,22 +151,29 @@ from the current package.
 In general it is best to be as specific as possible with your `InputSet`s,
 because all matching files will be provided to `declareOutputs`.
```

Comment:
`declareOutputs` doesn't exist any more :), but this is still a good idea. Maybe just change it to `because all matching files will be checked against your buildExtensions`? We still haven't talked about the buildExtensions field at this point though, but could link to that section?

Reply:
Done.